examples/api200/unmanaged_device.rb

```diff
@@ -11,5 +11,35 @@
 
 require_relative '../_client' # Gives access to @client
 
+def print_device(device)
+  puts "- \"#{device['name']}\" with model \"#{device['model']}\" and uri \"#{device['uri']}\""
+end
+
+puts "\nAdding first device ..."
+device1 = OneviewSDK::UnmanagedDevice.new(@client, name: 'Unmanaged Device 1', model: 'Procurve 4200VL')
+device1.add
+puts 'Device added:'
+print_device(device1)
```

Comment:
Change all instances of 
``print_device(device1)`` 
for something like
``puts "- \"#{device['name']}\" with model \"#{device['model']}\" and uri \"#{device['uri']}\""``